recipes/deploy-admin.rb

```diff
@@ -82,6 +82,7 @@
     install_multitenancy_config(most_recent_deploy, public_admin_hostname, public_engage_hostname)
     remove_felix_fileinstall(most_recent_deploy)
     install_smtp_config(most_recent_deploy)
+    install_default_tentant_config(most_recent_deploy, public_admin_hostname, private_hostname)
```

Comment:
LTI on the admin provides alternative access to processing status, scheduling, creating series, etc. I'll fix to deploy to workers so the file is the one source for all the nodes.